doc/api/cli.md

```diff
@@ -1410,7 +1410,8 @@ or poisoning attack. Avoid using this option.
 added: v6.3.0
 -->
 
-Activate inspector on `host:port`. Default is `127.0.0.1:9229`.
+Activate inspector on `host:port`. Default is `127.0.0.1:9229`. If port `0` is
```

Comment:
This behavior is applied to the other flags: `--inspect-port, --inspect-wait`

Reply:
Thanks, I've fixed it now